packages/itwin/ec3-widget/src/components/LabelAction.tsx

```diff
@@ -106,15 +102,15 @@ export const LabelAction = ({ template, onClose, label, setTemplate }: LabelActi
       setName(label.name);
       setItemName(label.elementNameColumn);
       setItemQuantity(label.elementQuantityColumn);
-      setMaterials(label.materials.map((x) => { return { nameColumn: x.nameColumn }; })); // creating a copy of array, so original (in the parent) isn't modified
+      setMaterials(label.materials.map((x) => x)); // creating a copy of array, so original (in the parent) isn't modified
```

Comment:
I think `[...label.materials]` would result in the same thing.

Reply:
Good point. Done.